Subject: [PATCH] add stdexcept
---
Index: bibcxx/Utilities/ConvertibleValue.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bibcxx/Utilities/ConvertibleValue.h b/bibcxx/Utilities/ConvertibleValue.h
--- a/bibcxx/Utilities/ConvertibleValue.h	(revision 179508b6116cd2688a224e619049c2108fbc001e)
+++ b/bibcxx/Utilities/ConvertibleValue.h	(date 1676142939920)
@@ -1,5 +1,6 @@
 #ifndef CONVERTIBLEVALUE_H_
 #define CCONVERTIBLEVALUE_H_
+#include <stdexcept>
 
 /**
  * @file ConvertibleValue.h
